spec/support/upload_params.cr

```diff
@@ -4,6 +4,7 @@ class Avram::UploadParams < Avram::Params
   @uploads : Hash(String, Avram::UploadedFile) = {} of String => Avram::UploadedFile
 
   def initialize(@uploads)
+    @hash = {} of String => Array(String)
```

Comment:
The specs wouldn't compile without this... How it was working before is magic....